react-backend/database/dbBuild.sql

```diff
@@ -1,67 +1,73 @@
 BEGIN;
-
-DROP TABLE IF EXISTS users CASCADE;
-DROP TABLE IF EXISTS visits CASCADE;
-DROP TABLE IF EXISTS activities CASCADE;
-DROP TABLE IF EXISTS cbusiness CASCADE;
-
-CREATE TABLE users (
+  DROP TABLE IF EXISTS users
+  CASCADE;
+DROP TABLE IF EXISTS visits
+CASCADE;
+DROP TABLE IF EXISTS activities
+CASCADE;
+DROP TABLE IF EXISTS cbusiness
+CASCADE;
+CREATE TABLE users
+(
   id SERIAL PRIMARY KEY,
   fullName VARCHAR(100) NOT NULL,
   sex VARCHAR(30) NOT NULL,
   yearOfBirth INTEGER NOT NULL,
   email VARCHAR(100),
+  date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   hash VARCHAR(64) NOT NULL
 );
-
-CREATE TABLE cbusiness (
+CREATE TABLE cbusiness
+(
   id SERIAL PRIMARY KEY,
   org_name VARCHAR(100) NOT NULL,
   genre VARCHAR(50) NOT NULL,
   email VARCHAR(100) NOT NULL,
```

Comment:
to make email unique you can change this line to:
```SQL
email VARCHAR(100) NOT NULL UNIQUE,
```
I assume the current method works too but I think this one is a little clearer?